src/adapters/scene-manager.ts

```diff
@@ -8,31 +8,37 @@ export async function createSceneManagerComponent(
   const { worlds, lands, sceneAdminManager } = components
 
   const { hasWorldOwnerPermission, hasWorldStreamingPermission, hasWorldDeployPermission } = worlds
-  const { hasLandUpdatePermission } = lands
+  const { getLandUpdatePermission } = lands
 
   async function isSceneOwner(place: PlaceAttributes, address: string): Promise<boolean> {
     const isWorlds = place.world
     if (isWorlds) {
       return await hasWorldOwnerPermission(address, place.world_name!)
     }
 
-    return await hasLandUpdatePermission(address, place.positions)
+    return (await getLandUpdatePermission(address, place.positions)).owner
   }
 
-  async function isSceneOwnerOrAdmin(place: PlaceAttributes, address: string): Promise<boolean> {
+  async function resolveUserScenePermissions(place: PlaceAttributes, address: string): Promise<UserScenePermissions> {
```

Comment:
What do you think about naming this function:
```suggestion
  async function getUserScenePermissions(place: PlaceAttributes, address: string): Promise<UserScenePermissions> {
```